README.md

```diff
@@ -0,0 +1,163 @@
+![Stream Cli](https://i.imgur.com/H8AScTq.png)
```

Comment:
not related to this PR but we probably want to have a new logo for this

Reply:
sure. maybe something with the Gopher character :)

https://go.dev/blog/gopher

Reply:
Love it! A Pirate Gopher on top of the Stream boat